### ISSUE TITLE
The name is confusing

### ISSUE THREAD

**Issue by boland25** [kmustahsan/Swiftly#3]
You are writing an Android application but the name is Swift-ly, which suggests that the code is written in this most amazing of languages called Swift.

**Comment by thesaadismail** [comment]
👍 Agreed, I feel like the description needs additional details as well. What does it mean to generate a shopping cart?

**Comment by kmustahsan** [comment]
@boland25  Hahaha. Our team determined programming for iOS using Swift was far better than programming in Android, so we paid homage to the best language of all.

**Comment by kmustahsan** [comment]
@thesaadismail Good point! Initially we left out any descriptions so our idea wouldn't be stolen... and then Amazon Go happened. So now we're trying to get credit for Go. :)
